geometry_msgs/TransformStamped hand_base_tf
---
geometry_msgs/TransformStamped camera_base_tf
std_msgs/Bool success